src/components/MultiHopTrade/hooks/useGetTradeQuotes.tsx

```diff
@@ -84,67 +67,5 @@ export const useGetTradeQuotes = () => {
     wallet,
   ])
 
-  const lifiQuery = useGetLifiTradeQuoteQuery(debouncedTradeQuoteInput, {
-    skip: !flags.LifiSwap,
-  })
-
-  const thorQuery = useGetThorTradeQuoteQuery(debouncedTradeQuoteInput, {
-    skip: !flags.ThorSwap,
-  })
-
-  // TODO(woodenfurniture): quote selection
-  const sortedQuotes: TradeQuoteResult[] = useMemo(() => {
```

Comment:
All of this now happens inside the RTK query, which matches what the end-state API will do (return sorted quotes).